sbol_utilities/component.py

```diff
@@ -60,6 +60,26 @@ def check_roles(component: sbol3.Component) -> bool:
         and has_dna_type(obj) and len(obj.sequences) == 1
 
 
+def by_roles(obj: sbol3.TopLevel, required_role: str) -> bool:
+    """Search the entire provided document, and return components which have specified role as one of its roles.
```

Comment:
This description isn't quite correct, nor the one for by_types

Reply:
Fixed, I forgot to change these while changing functions